src/printer.rs

```diff
@@ -447,7 +447,7 @@ impl<'a> Printer for InteractivePrinter<'a> {
             };
             highlighter_from_set
                 .highlighter
-                .highlight(&line, highlighter_from_set.syntax_set)
+                .highlight_line(&line, highlighter_from_set.syntax_set)?
```

Comment:
Do we really want to bubble up errors here? That would make bat exit.. and we would not show the end of the file, right?

Don't get me wrong. It's obviously better than panicking. I'm just trying to understand if there are reasonable error cases here that we would want to handle locally.

Reply:
@sharkdp Good point. Not sure yet exactly what to do other than bubble up though, and in what cases.

Since this PR is a pure improvement in this regard (erroring instead of panicing) I propose that we merge and release this as-is, and then incrementally improve the situation over time, once we figure out what to do instead. Sounds OK?

Reply:
Yes - thank you